modules/fluentd/templates/fluent.conf

```diff
@@ -69,12 +69,16 @@
     @type elasticsearch
     host ${elasticsearch_hostname}
     scheme https
-    logstash_format true
-    logstash_prefix "$${tag}"
     port ${elasticsearch_port}
     flush_interval 5s
     include_tag_key true
 
+    logstash_format true
+    logstash_prefix "$${tag}"
+    {{ if eq (keyOrDefault "${weekly_index_enabled_consul_key}" "false") "true" }}
+    logstash_dateformat %G.W%V
```

Comment:
Apologies, this was supposed to be `logstash_dateformat`.

References:
https://github.com/uken/fluent-plugin-elasticsearch#logstash_dateformat
https://apidock.com/ruby/Time/strftime